README.md

```diff
@@ -1 +1,95 @@
-# cars-bnb-frontend
\ No newline at end of file
+# Cars BnB App
```

Comment:
- [ ] On your backend, you are using Rspec, but the "test" folder still exists which is used by another test framework.  Kindly delete that folder if not using it.